src/Factory/MaintenanceConfigurationFactory.php

```diff
@@ -46,6 +46,7 @@ public static function map(MaintenanceConfiguration $maintenanceConfiguration, ?
             'custom_message' => '',
             'token' => '',
             'allow_bots' => false,
+            'allow_admins' => false,
```

Comment:
if you add such property you have to update phpdoc block under at [line 59](https://github.com/synolia/SyliusMaintenancePlugin/pull/43/files#diff-e2463c552289188a9af1dfc531bff17dbb07d2217d7a05c1f3afd46442ae8cedR59)

```
         /**
          * @var array{
          *     'ips': array<string>,
          *     'channels': array<string>,
          *     'scheduler': array{'start_date': string, 'end_date': string},
          *     'custom_message': string,
          *     'token': string,
          *     'allow_bots': bool,
>         *     'allow_admins': bool,
          *     'enabled': bool,
          *  } $options
          */
```